e2e/cmd/e2e/pkg/sql.go

```diff
@@ -77,6 +79,11 @@ func ParseRows(rows *sql.Rows) (outputRows []interface{}, err error) {
 				continue
 			}
 
+			if z, ok := (scanArgs[i]).(*sql.NullTime); ok {
+				row[v.Name()] = z.Time
+				continue
+			}
+
```

Comment:
If we write this, then we cannot distinguish between null and zero value. We should output a JSON null if the value is a SQL NULL.